contracts/utils/MultiSendERC20.sol

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.24;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+
+contract MultiSendERC20 is Ownable {
+  uint8 public arrayLimit;
+  ERC20 public token;
+
+  event Multisended(uint256 total, address token);
+
+  constructor(address _owner, address _token) Ownable(_owner) {
+    arrayLimit = 200;
+    token = ERC20(_token);
+  }
+
+  function setArrayLimit(uint8 _arrayLimit) external onlyOwner {
+    arrayLimit = _arrayLimit;
+  }
+
+  function multisendToken(address[] calldata _recipients, uint256[] calldata _amounts) public {
```

Comment:
Don't you want this to be called only by the `owner`?

Reply:
No need, since the tokens are sent from the message owner we don't need that control. The only thing I want to control is the limit of how many can be sent at a time